Makefile

```diff
@@ -7,7 +7,10 @@ run-postgres:
 	$(CONTAINER_RUNTIME) run -it -v data:/var/lib/postgresql/data -e POSTGRES_USER=kai -e POSTGRES_PASSWORD=dog8code -e POSTGRES_DB=kai -p 5432:5432 docker.io/pgvector/pgvector:pg15
 
 run-server:
-	PYTHONPATH=$(KAI_PYTHON_PATH) python ./kai/server.py
+	PYTHONPATH=$(KAI_PYTHON_PATH) python ./kai/server.py --loglevel "info"
+
+run-server-debug:
+	PYTHONPATH=$(KAI_PYTHON_PATH) python ./kai/server.py --loglevel "debug"
 
 load-data:
 	PYTHONPATH=$(KAI_PYTHON_PATH) python ./kai/incident_store_advanced.py --config_filepath ./kai/database.ini
```

Comment:
Looks the database.ini doesnt exist anymore. change to 
```
PYTHONPATH=$(KAI_PYTHON_PATH) python ./kai/incident_store_advanced.py --config_filepath ./kai/config.toml

Reply:
Thank you @savitharaghunathan for catching that!